app/extensions/brave/locales/en-US/error.properties

```diff
@@ -1,9 +0,0 @@
-errorReload=Retry the URL
```

Comment:
Was removing this file intended? It's still referenced in a few places:
https://github.com/brave/browser-laptop/blob/e691f7624f85e36045f6728c4735b1e5e4a0dde0/app/locale.js#L384
https://github.com/brave/browser-laptop/blob/e691f7624f85e36045f6728c4735b1e5e4a0dde0/app/extensions/brave/about-contributions.html#L19
https://github.com/brave/browser-laptop/blob/e691f7624f85e36045f6728c4735b1e5e4a0dde0/app/extensions/brave/about-error.html#L17
https://github.com/brave/browser-laptop/blob/e691f7624f85e36045f6728c4735b1e5e4a0dde0/app/extensions/brave/about-preferences.html#L19

(possibly others)